net-analyzer/ostinato/ostinato-1.3.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+DESCRIPTION="Packet generator and analyzer"
+HOMEPAGE="https://ostinato.org/"
+SRC_URI="https://github.com/pstavirs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+#https://github.com/pstavirs/ostinato/issues/173
+# libpcap dep is versioned to pull in the fix for #602098
+RDEPEND="
+	dev-libs/protobuf:=
```

Comment:
<dev-libs/protobuf-22.5:= 

See https://bugs.gentoo.org/912829 and https://bugs.gentoo.org/913106

Reply:
seems to build with glibc 2.38